### PR TITLE
XGBoost tutorial

### PR DESCRIPTION
**Pull request by Yancey0623**

### CODE REVIEW

File: example/jupyter/tutorial_xgboost.ipynb [hunk 1]
```diff
@@ -0,0 +1,357 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# XGBoost on SQLFlow Tutorial\n",
+    "\n",
+    "This is a tutorial on train/predict XGBoost model in SQLFLow, you can find the design doc from [here](https://github.com/sql-machine-learning/sqlflow/blob/develop/doc/xgboost_on_sqlflow_design.md), in this tutorial you will know how to:\n",
```
Comment (tonyyang-svail) [inline example/jupyter/tutorial_xgboost.ipynb]: Instead of linking the design doc, linking the user guide might be more useful.

**Review by tonyyang-svail — DISMISSED** [review]
LGTM.

File: example/jupyter/tutorial_xgboost.ipynb (hunk 1 above)
Comment (Yancey0623) [reply example/jupyter/tutorial_xgboost.ipynb]: done

Comment (typhoonzero) [inline example/jupyter/tutorial_xgboost.ipynb]: no need to mention design in a tutorial?

File: example/jupyter/tutorial_xgboost.ipynb [hunk 2]
```diff
@@ -0,0 +1,357 @@
...
+    "- Train a XGBoost model to fit the boston housing price; and\n",
```
Comment (typhoonzero) [inline example/jupyter/tutorial_xgboost.ipynb]: Train a XGBoost model to fit the boston housing dataset

File: example/jupyter/tutorial_xgboost.ipynb (hunk 1 above)
Comment (typhoonzero) [reply example/jupyter/tutorial_xgboost.ipynb]: "you will know how to" -> learn how to...

File: example/jupyter/tutorial_xgboost.ipynb [hunk 3]
```diff
@@ -0,0 +1,357 @@
...
+    "age|proportion of owner-occupied units built prior to 1940.\n",
+    "dis|weighted mean of distances to five Boston employment centres.\n",
+    "rad|index of accessibility to radial highways.\n",
+    "tax|full-value property-tax rate per \\$10,000.\n",
+    "ptratio|pupil-teacher ratio by town.\n",
+    "black|1000(Bk - 0.63)^2 where Bk is the proportion of blacks by town.\n",
+    "lstat|lower status of the population (percent).\n",
+    "medv|median value of owner-occupied homes in $1000s.\n",
+    "\n",
+    "We seperated the dataset into train/test datasets which used to train/predict our model."
```
Comment (typhoonzero) [inline example/jupyter/tutorial_xgboost.ipynb]: SQLFlow will auto split train/validation dataset, shall we mention this?

File: example/jupyter/tutorial_xgboost.ipynb (hunk 1 above)
Comment (Yancey0623) [reply example/jupyter/tutorial_xgboost.ipynb]: > no need to mention design in a tutorial?

Yep, already linked the user guide doc.

Comment (Yancey0623) [reply example/jupyter/tutorial_xgboost.ipynb]: done.

File: example/jupyter/tutorial_xgboost.ipynb (hunk 2 above)
Comment (Yancey0623) [reply example/jupyter/tutorial_xgboost.ipynb]: done.

File: example/jupyter/tutorial_xgboost.ipynb (hunk 3 above)
Comment (Yancey0623) [reply example/jupyter/tutorial_xgboost.ipynb]: Good idea, fixed.

**Review by typhoonzero — APPROVED** [review]

**Pull request merged**